pytest.ini

```diff
@@ -1,6 +1,5 @@
 [pytest]
-addopts = --pyargs idaes
-          --durations=100
+addopts = --durations=100
```

Comment:
Is it fine to push this change? I think the test file will fail without it since the Excel could not be imported.